src/ctia/properties.clj

```diff
@@ -198,16 +198,9 @@
                       "ctia.migration.batch-size" s/Int
                       "ctia.migration.buffer-size" s/Int
 
-                      "ctia.store.asset" s/Str
```

Comment:
it's not related to this PR, but I discovered it. there is a function in the same ns which automatically generates it.